packages/openapi-generator/src/generator.ts

```diff
@@ -201,16 +201,16 @@ async function generateFromFile(
  * @param input the path to the input directory.
  * @returns all file paths as a string array.
  */
-async function getInputFilePaths(input: string): Promise<string[]> {
+export async function getInputFilePaths(input: string): Promise<string[]> {
   if ((await lstat(input)).isFile()) {
     return [input];
   }
 
-  const directoryContents = await readdir(input, { withFileTypes: true });
+  const directoryContents = await readdir(input);
```

Comment:
I now wonder why nothing is failing. This should not work... Because later execution needs the file type...

Reply:
Well, previously the filetype was derived from the `fs.Dirent` object I believe, but since then `lstat()` was used, so we have a stats object already, which is why it is not failing.

Reply:
Also, `lstat` was used instead of the `Dirent`, because of consistency if I remember